diff --git a/cmake/FindJournald.cmake b/cmake/FindJournald.cmake
index 874d8ba..39bf54c 100644
--- a/cmake/FindJournald.cmake
+++ b/cmake/FindJournald.cmake
@@ -15,7 +15,7 @@
 # use pkg-config to get the directories and then use these values
 # in the FIND_PATH() and FIND_LIBRARY() calls
 find_package(PkgConfig)
-pkg_check_modules(PC_JOURNALD QUIET libsystemd-journal)
+pkg_check_modules(PC_JOURNALD QUIET systemd)
 
 set(JOURNALD_FOUND ${PC_JOURNALD_FOUND})
 set(JOURNALD_DEFINITIONS ${PC_JOURNALD_CFLAGS_OTHER})
